tests/test_dagmc_file.py

```diff
@@ -77,6 +80,88 @@ def test_set_entityset_ranges():
             type_range == single_cube.entityset_ranges[set_type])
     assert(all(test_pass))
 
+def test_set_dimension_meshset():
+    """
+    Tests different aspects of the set_dimension_meshset function
+    """
+    single_cube = ds.DagmcStats(test_env[1]['input_file'])
+    range = single_cube._my_moab_core.get_entities_by_type(
+                            single_cube.root_set, types.MBENTITYSET)
+    assert(set(single_cube.dim_dict.values())==set(range[-4:]))
```

Comment:
I found that the dictionary elements are not in the insertion order for adding key-value pairs to the empty dictionary, so I used set() == set() here. Or should I sort the dictionary and check if the elements are in the insertion order?